src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -234,22 +234,18 @@ public boolean compareAndSwapLong(@NonNegative long offset, long expected, long
     @Override
     public @NotNull AbstractBytes<U> append(double d)
             throws BufferOverflowException, IllegalStateException {
-        boolean fits = canWriteDirect(380);
-        double ad = Math.abs(d);
-        if (ad < 1e-18) {
-            append(Double.toString(d));
-            return this;
-        }
-        if (!fits) {
-            fits = 1e-6 <= ad && ad < 1e20 && canWriteDirect(24);
-        }
+        boolean fits = canWriteDirect(32);
```

Comment:
Is 32 sufficient? is it because we put exponents in for really large/small numbers?